community/asking-questions.md

```diff
@@ -0,0 +1,12 @@
+# Asking Questions on the Forum
+
+Author(s): code3z (@code3 on the forum)
+
+The [modding forum](https://forum.vivaldi.net/category/52/modifications) is also a great place to ask questions about modding. Before asking, search the forum and the internet to make sure there is not already a good answer.
```

Comment:
~~make sure~~ → check if

(‘make sure’ sounds like you’ll kill them if there’s an answer *somewhere*)

Reply:
> ‘make sure’ sounds like you’ll kill them if there’s an answer somewhere

I mean, I never said I wouldn’t 🤷‍♂️

I’ll fix that.